From aae7895b774d7d96665ae20aeecd159368439c01 Mon Sep 17 00:00:00 2001
From: Rafael Palomar <rafael.palomar@ous-research.no>
Date: Sat, 21 Feb 2026 21:18:45 +0100
Subject: [PATCH] ENH: Add MRMLCore dependency to Widgets target libraries

This adds the missing MRMLCore library to the Models Widgets
CMakeLists target libraries list, resolving a potential linking issue.
---
 Modules/Loadable/Models/Widgets/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/Loadable/Models/Widgets/CMakeLists.txt b/Modules/Loadable/Models/Widgets/CMakeLists.txt
index 3a24095a6c..ede02115e7 100644
--- a/Modules/Loadable/Models/Widgets/CMakeLists.txt
+++ b/Modules/Loadable/Models/Widgets/CMakeLists.txt
@@ -26,6 +26,7 @@ set(${KIT}_RESOURCES
 
 set(${KIT}_TARGET_LIBRARIES
   qSlicerSubjectHierarchyModuleWidgets
+  MRMLCore
   )
 
 #-----------------------------------------------------------------------------
-- 
2.52.0

